### ISSUE TITLE
VidCoder won't start

### ISSUE THREAD

**Issue by trecoool** [RandomEngy/VidCoder#1231]
### Problem Description

Hello

Vidcoder won't start anymore, up-to-date, portable or setup.

Problem seems to be with my VidCoder.sqlite file : if i delete it then vidcoder will start, but then i loose my precious presets...

Any way to get the presets back?

Thank you

### What version of VidCoder are you running?

9.20

**Comment by RandomEngy** [comment]
Does it crash with a specific message? If you share the VidCoder.sqlite file, I can take a look and see if I can figure out what went wrong and either fix the app or repair the file.

**Comment by trecoool** [comment]
It does not crash. It seems to startup, but no UI shows up. It just shows in the taskbar, and I'm able to right click on it and close it, so it does not seem to be in a crashed state. Here is the sqlite file! Thank you 🙏
[VidCoder sqlite.zip](https://github.com/RandomEngy/VidCoder/files/15109252/VidCoder.sqlite.zip)

**Comment by RandomEngy** [comment]
Interesting, VidCoder will open for me with that VidCoder.sqlite file. What version of Windows are you using? I saw that the selected picker is saving to a file path of "./Stream", so I changed it to some other absolute path to see if that would fix it for you:

[VidCoder-updated.sqlite.zip](https://github.com/RandomEngy/VidCoder/files/15111818/VidCoder-updated.sqlite.zip)

**Comment by trecoool** [comment]
I'm on windows 10 pro 21H2 19044.3086.
Your fix does it! It's now starting again.
The "./Stream" was an attempt from me to try to export in current folder in a "stream" folder.
Interesting that this was the issue.
If there is a way to do that that would be awesome!
Let me know if I can do anything else to help with that issue.

Thanks a lot for your help!
I love your piece of software, big kudos!

**Comment by RandomEngy** [comment]
You'd use "Custom file naming format" `Stream\{source}`. You can optionally combine with "Output to source folder if possible".

**Comment by trecoool** [comment]
Ah this is great. I just went to try that, and actually there is still an issue with a ui not showing up, this time it is the window of the encoding setting. Picker works. Encoding setting works if I delete the sqlite file. Here's a video showing what it looks like 

https://github.com/RandomEngy/VidCoder/assets/49880932/06471748-dd46-4495-8bf3-42b4befbdc14

**Comment by RandomEngy** [comment]
Hmm. Do you have a second monitor? What do you see in Display Settings? I am guessing that window is off-screen. I have code that's supposed to move it back to the closest visible monitor. Either you have a monitor that is not visible/enabled, or the code is not working correctly.

**Comment by RandomEngy** [comment]
If you don't have any monitors listed in Display Settings that are not visible to you, I can add some more logging in that code to figure out what's going on.

**Comment by trecoool** [comment]
I have an additional display, it does not show up on it. When i click on the encoding settings, the new window should be focused and I should be able to bring it back with win+arrows? It works with other windows (they show up and i can move them with win+arrows), so unsure. No problem for the logging!

**Comment by RandomEngy** [comment]
Sorry for the delayed response here. I added some logging to 10.3 Beta. You need to enable it under Global options -> Advanced -> Logging verbosity -> Extended.

If you have it start up without the encoding window, look at the General tab and send the logs you see it write there on startup.

**Comment by trecoool** [comment]
No worries about the delay! Thank you for looking into it.
I downloaded the 10.3 and same issue happens. I enabled the extended logging, restarted the app, tried to click on the encoding window, then closed the app. Here's the log file that I got :

`VC [15:10:08] VidCoder 10.3 Beta
VC [15:10:09] Placing VidCoder.ViewModel.MainViewModel
VC [15:10:09] Applying placement: {"length":44,"flags":0,"showCmd":1,"minPosition":{"X":-1,"Y":-1},"maxPosition":{"X":-1,"Y":-1},"normalPosition":{"Left":2214,"Top":467,"Right":3360,"Bottom":1325}}
VC [15:10:09] closestMonitorInfo - rcMonitor: {"Left":0,"Top":0,"Right":3440,"Bottom":1440}, rcWork: {"Left":0,"Top":0,"Right":3440,"Bottom":1410}
VC [15:10:09] Placing VidCoder.ViewModel.EncodingWindowViewModel
VC [15:10:09] Applying placement: {"length":44,"flags":0,"showCmd":1,"minPosition":{"X":-1,"Y":-1},"maxPosition":{"X":-1,"Y":-1},"normalPosition":{"Left":-1040,"Top":755,"Right":0,"Bottom":1532}}
VC [15:10:09] closestMonitorInfo - rcMonitor: {"Left":0,"Top":0,"Right":3440,"Bottom":1440}, rcWork: {"Left":0,"Top":0,"Right":3440,"Bottom":1410}
VC [15:10:27] Applying placement: {"length":44,"flags":0,"showCmd":1,"minPosition":{"X":-1,"Y":-1},"maxPosition":{"X":-1,"Y":-1},"normalPosition":{"Left":2210,"Top":547,"Right":2885,"Bottom":1096}}
VC [15:10:27] closestMonitorInfo - rcMonitor: {"Left":0,"Top":0,"Right":3440,"Bottom":1440}, rcWork: {"Left":0,"Top":0,"Right":3440,"Bottom":1410}
`
